core/modules/simpletest/simpletest.module

```diff
@@ -539,6 +576,45 @@ function simpletest_clean_results_table($test_id = NULL) {
   return 0;
 }
 
+
+/**
+ * Removes cached profile tables from the database.
+ */
+function simpletest_clean_profile_cache_tables(){
+  $tables = db_find_tables(Database::getConnection()->prefixTables('{simpletest_cache_}') . '%');
+  $count = 0;
```

Comment:
Instead of using `$count = 0;` and `$count++` here, you could do something like `$count = count($tables);`.